web/src/components/userpage/event-review-card.js

```diff
@@ -1,34 +1,48 @@
 import {
-  Card, 
-  CardContent, 
-  Grid, 
-  Typography
+  Grid,
+  Typography,
 } from '@material-ui/core';
 import EventReview from './event-review';
 import EventSchedule from './event-schedule';
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
+import React, {Component, Fragment} from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
 import {withRouter} from 'react-router-dom';
 
 class EventReviewCard extends Component {
   constructor(props) {
     super(props);
   }
 
+  parseDateToString(date) {
+    const monthNames = ['January', 'February', 'March', 'April', 'May',
+      'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
kinda ugly. Another way to do this is something like:

const date = new Date(2009, 10, 10); // 2009-11-10
const month = date.toLocaleString('default', { month: 'long' });

https://stackoverflow.com/questions/1643320/get-month-name-from-date